README.md

```diff
@@ -25,12 +28,79 @@ pip install -r requirements.txt
 # be ready to create the status database. By installing the package above
 # you should have access to two command-line commands, one of which
 # will create the database for you
-create_monitoring_db <path/to/status_database.db>
+create_monitoring_db <path/to/new_status_database.db>
 # note, if you need to destroy and recreate the status data at any point
 # you can do so with:
 # create_monitoring_db <path/to/status_database.db> --drop
 ```
 
+### Sample Job
+The following commands will create a database in which to store ETL job statuses and insert data from example.csv to a CKAN instance running at http://data.wprdc.org.
+
+```bash
+cd /path/to/jobs_dir/
+# create a status tracking sqlite db called status.db in jobs_dir/
+create_monitoring_db status.db
+# note, if you need to destroy and recreate the status data in status.db
+# at any point, you can do so with:
+# create_monitoring_db status.db --drop
+# run example_pipeline from example_job.py in jobs_dir/
+run_job example_job:example_pipeline
+```
+
+##### example.csv:
+| name      | birthdate  | last_visit       | visit_count |
+|-----------|------------|------------------|-------------|
+| Susan     | 03/09/2016 | 01/25/2016T03:07 | 21          |
+| Xenos     | 10/29/2015 | 01/01/2016T22:41 | 9           |
+| Jordan    | 04/27/2016 | 10/25/2015T22:14 | 50          |
+| Xander    | 03/21/2017 | 06/19/2016T12:59 | 40          |
+
+##### example_job.py:
+```python
+import os
+import datetime
+from marshmallow import fields, pre_load
+import pipeline as pl
+
+class ExampleSchema(pl.BaseSchema):
+    name = fields.String()
+    birthdate = fields.Date(format='%m/%d/%Y')
+    last_visit = fields.DateTime(format='%m/%d/%YT%H:%M')
+    visit_count = fields.Integer()
+
+    class Meta:
+        ordered=True
+
+    @pre_load()
+    def format_date(self, data):
+        data['birthdate'] = datetime.datetime.strptime(data['birthdate'],'%m/%d/%Y').date().isoformat()
+
+target = os.path.dirname(os.path.realpath(__file__)) + "/example.csv"       # target file from which to extract data (in this case, it's a local file)
```

Comment:
This is ok for now, but I think that in the future it would be good to figure out a way to avoid having to specify this for every job.

I feel like these are properties of the pipeline object itself, not necessarily arguments for the loaders. I wonder what the right middle ground would be for this.